### ISSUE TITLE
Support for `where` parameter in Softmax

### ISSUE THREAD

**Issue by 932179209** [enpasos/jax2onnx#138]
Right now, `SoftmaxPlugin` doesn't accept the `where` parameter from `jax.nn.softmax()`. This breaks when converting any model that uses masked softmax:

```python
jax.nn.softmax(x, axis=-1, where=mask) 
```

This is a blocker for pretty much all Transformer models (BERT, GPT, etc.) since they rely on masked attention for handling padding tokens.

Hope it can be supported in the future!

**Comment by enpasos** [comment]
Quick update: `jax.nn.softmax` with a `where` mask is now supported in the converter. The plugin binds the mask, lowers it by writing `-inf` into masked positions before the ONNX `Softmax`, zeros masked outputs afterward, and handles batched/`vmap` cases. A regression testcase for masked softmax is in place. If you pull the latest `main` (or the next release), masked attention paths (e.g. BERT/GPT) should convert without breaking.
